### DISCUSSION TITLE
Touchpad Scroll and gestures direction.

### DISCUSSION THREAD

**Discussion by arshali2774** [prasanthrangan/hyprdots#1965] [Q&A]
Hi, first of all I love the HyDE. Just switched from windows 11. 
Can someone tell me how to change the direction of the scroll on touchpad of laptop. Also the gestures direction are also inversed.
In windows we scroll up to go down and vice versa. However mouse scroll wheel is working fine.
The gestures on browser to go back and forward are swipe right and swipe left respectively.
In summary just want to reverse the current scroll and gesture directions.

**Comment by kRHYME7** [comment]
Reference your configs from this
 https://wiki.hyprland.org/Configuring/Variables/#touchpad
https://wiki.hyprland.org/Configuring/Variables/#gestures
put this in you `~/.config/hypr/userprefs.conf ` 

it should look like this 

```hyprlang

gestures {
workspace_swipe_invert=false
workspace_swipe_fingers=3 # or make it 4 if needed
}

```
